src/frontend/components/SelectContent/index.spec.tsx

```diff
@@ -0,0 +1,323 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+import { Grommet } from 'grommet';
+
+import { appData } from '../../data/appData';
+import {
+  documentMockFactory,
+  videoMockFactory,
+} from '../../utils/tests/factories';
+import { SelectContent } from './index';
+import { wrapInIntlProvider } from '../../utils/tests/intl';
+import { uploadState } from '../../types/tracks';
+
+jest.mock('../../data/appData', () => ({
+  appData: {
+    new_document_url: 'https://example.com/lti/documents/new-hash',
+    new_video_url: 'https://example.com/lti/videos/new-hash',
+    lti_select_form_action_url: '/lti/select/',
+    lti_select_form_data: {},
+  },
+}));
+
+jest.mock('../Loader', () => ({
+  Loader: () => <span>Loader</span>,
+}));
+
+window.HTMLFormElement.prototype.submit = jest.fn();
+
+describe('<SelectContent />', () => {
+  afterEach(jest.resetAllMocks);
+
+  describe('content', () => {
+    it('displays content infos', async () => {
+      render(
+        wrapInIntlProvider(
+          <Grommet>
+            <SelectContent
+              documents={[
+                documentMockFactory({
+                  id: '1',
+                  title: 'Document 1',
+                  upload_state: uploadState.PROCESSING,
+                  is_ready_to_show: false,
+                }),
+              ]}
+              videos={[
+                videoMockFactory({
+                  id: '1',
+                  title: 'Video 1',
+                  upload_state: uploadState.PROCESSING,
+                  is_ready_to_show: false,
+                }),
+                videoMockFactory({
+                  id: '2',
+                  title: 'Video 2',
+                  upload_state: uploadState.READY,
+                  is_ready_to_show: true,
+                }),
+              ]}
+              new_document_url={appData.new_document_url}
+              new_video_url={appData.new_video_url}
+              lti_select_form_action_url={appData.lti_select_form_action_url!}
+              lti_select_form_data={appData.lti_select_form_data!}
+            />
+          </Grommet>,
+        ),
+      );
+
+      screen.getByTitle('Select Video 1');
+      expect(
+        screen.getByTitle('Select Video 1').getElementsByTagName('img')[0],
+      ).toHaveAttribute('src', 'https://example.com/default_thumbnail/144');
+      fireEvent.mouseEnter(screen.getByTitle('Select Video 1'));
+      screen.getByText('Video 1');
```

Comment:
If this is expected after clicking, you should also test before clicking it does not exists.